lexos/templates/scrub.html

```diff
@@ -34,6 +34,7 @@
                 }
                                 });
            tagDict=JSON.stringify(tagDict);
+           console.log(tagDict);
```

Comment:
one of the plugin that will make it easier for you to pass the CI is `editorconfig`. we have an editor config file in the root of our project, this takes care of final new line and trailing white spaces as well as indentation